[2025-09-08T17:08:13.439+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-09-08T17:08:13.447+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskA backfill__2025-04-06T12:00:00+00:00 [queued]>
[2025-09-08T17:08:13.451+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskA backfill__2025-04-06T12:00:00+00:00 [queued]>
[2025-09-08T17:08:13.451+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-09-08T17:08:13.463+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): taskA> on 2025-04-06 12:00:00+00:00
[2025-09-08T17:08:13.469+0000] {standard_task_runner.py:72} INFO - Started process 19768 to run task
[2025-09-08T17:08:13.474+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'cron_catchup_backfill', 'taskA', 'backfill__2025-04-06T12:00:00+00:00', '--job-id', '119', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/executing_cron_catchup_backfill.py', '--cfg-path', '/tmp/tmpgd9ud5w9']
[2025-09-08T17:08:13.476+0000] {standard_task_runner.py:105} INFO - Job 119: Subtask taskA
[2025-09-08T17:08:13.509+0000] {task_command.py:467} INFO - Running <TaskInstance: cron_catchup_backfill.taskA backfill__2025-04-06T12:00:00+00:00 [running]> on host codespaces-9bd7b3
[2025-09-08T17:08:13.563+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='loonycorn' AIRFLOW_CTX_DAG_ID='cron_catchup_backfill' AIRFLOW_CTX_TASK_ID='taskA' AIRFLOW_CTX_EXECUTION_DATE='2025-04-06T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='backfill__2025-04-06T12:00:00+00:00'
[2025-09-08T17:08:13.564+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-09-08T17:08:13.577+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-09-08T17:08:13.577+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo TASK A has executed!']
[2025-09-08T17:08:13.584+0000] {subprocess.py:86} INFO - Output:
[2025-09-08T17:08:13.585+0000] {subprocess.py:93} INFO - TASK A has executed!
[2025-09-08T17:08:13.585+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-09-08T17:08:13.598+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-09-08T17:08:13.598+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=cron_catchup_backfill, task_id=taskA, run_id=backfill__2025-04-06T12:00:00+00:00, execution_date=20250406T120000, start_date=20250908T170813, end_date=20250908T170813
[2025-09-08T17:08:13.647+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-09-08T17:08:13.664+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-08T17:08:13.668+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
